mantidproject/mantid
# Engineering Diffraction GUI allow runs to be removed from the fitting list

The ability to remove runs from the list widget (and the model) was added to the ED GUI. Users can now discard runs they're not interested in before doing **Fit All**.

**To test:**

Make sure `EnggDiffFittingModelTest` and `EnggDiffFittingPresenterTest` pass locally.

Some sample data is [here](https://github.com/mantidproject/mantid/files/1460806/ENGINX_focused_runs.zip).

Load up the ED GUI and go to the **Fitting** tab.

Load at least 2 of the runs from the attached zip file. Select one, and click **Remove run**. The run should disappear from the list widget, and the corresponding workspace should no longer appear in the workspaces area.

Fixes #21365 

**Release Notes** 
Doesn't need to be in the release notes, as is quite a minor (and obviously necessary) part of the larger work to enable multirun fitting

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
